third_party/islet/setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# ##############################################################################
+# setup.sh - Setup script to build ISLET SDK
+# ##############################################################################
+
+set -Eeuo pipefail
```

Comment:
(FYI: I'm assuming you've run `shellcheck` on this script. Right now, there are only a few Bash scripts. I am thinking it will be good to invoke `shellcheck` as part of CI, so that every new script change will adhere to common formatting. 

That's not in-place, just yet ... but will be good if have run this thru `shellcheck` while we are adding a new script.
)

Reply:
Yes. It's very good to add `shellcheck` to CI. 
And  c++ conding style check on CI is also good too =)